docs/UserGuide.md

```diff
@@ -123,6 +147,9 @@ Format: `add_contact n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
 * A contact cannot be added if their phone number already exists
 * A contact cannot be added if their name already exists
 
+<box type="warning" seamless>
+
+**WARNING** : Please ensure there is no more than **one** whitespace in-between each word for the contact name!
 
```

Comment:
Maybe can also include that the `name ` should only consists of alphanumeric characters (either as a warning or include it in the notes)

Reply:
added